src/Box.js

```diff
@@ -1,11 +1,14 @@
 import styled from 'styled-components'
-import { space, width, color } from 'styled-system'
+import { space, width, color, responsiveStyle } from 'styled-system'
 import PropTypes from 'prop-types'
 
+const visibility = responsiveStyle('visibility', 'visibility')
```

Comment:
I'm guessing we'll actually want to use `display: none` for this since `visibility: hidden` keeps the element in the document flow (and still takes up space). The other consideration here is that the `responsiveStyle` function only supports mobile-first min-width media queries, whereas we might actually want to only hide between two breakpoints.